app/views/filter/_filter.html.erb

```diff
@@ -96,6 +96,15 @@
           </div>
         </div>
 
+        <div class="filter__group mt-1/4 mb-1/1">
+          <p class="filter__label">Players</p>
+          <%= content_tag :div, nil, class: "num-player-slider", data: {
```

Comment:
I've personally been using `tag.div` rather than `content_tag`, I think for the sake of consistency this one should be changed.